src/cli/deploy.ts

```diff
@@ -45,6 +63,8 @@ export async function deploy(opts: { prefix?: string, customDomain?: string }):
 
     console.log("Uploading game...");
 
+    const files = await walkPublishDir(PUBLISH_DIR);
+    console.log(`Archiving ${kleur.bold(files.length)} files...`);
```

Comment:
Let's avoid using `kleur.bold()` for now, @joshplayco reported that it was causing text to render as invisible in his terminal.